x-pack/plugins/actions/server/sub_action_framework/helpers/validators.ts

```diff
@@ -9,6 +9,23 @@ import { i18n } from '@kbn/i18n';
 import { get } from 'lodash';
 import { ValidatorServices } from '../../types';
 
+const validProtocols: string[] = ['http:', 'https:'];
+export const assertURL = (url: string) => {
```

Comment:
As this function is the same as in `x-pack/plugins/actions/server/sub_action_framework/sub_action_connector.ts` can we use the function from the helpers in `sub_action_connector.ts`. Also, can you add a test for this new validator?